rio_tiler_pds/cbers/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""rio-tiler-pds.cbers"""
+
+from .awspds_cbers import AWSPDS_CBERSReader  # noqa
```

Comment:
What about having an `aws` module, so that you don't have to prefix all the classes with `AWSPDS`?

Like
```
rio_tiler_pds.aws.sentinel2
rio_tiler_pds.aws.cbers
rio_tiler_pds.aws.landsat8
```

Reply:
sure!